src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar-wrapper {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  padding: $calendar-padding;
+
+  @each $day, $number in $days-of-week {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($cell-size + $cell-gap) * $number;
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} :nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square($cell-size);
+    background-color: $cell-color;
+    border: 1px solid black;
+    font-size: 30px;
+    text-align: center;
+    transition: color 0.5s, transform 0.5s;
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/235866270-dca5b0bb-1541-47dc-8380-8806d934580a.png)

Reply:
you shouldn't remove it at all, just move to the main selector